### PR TITLE
Fix Antenna template loading for Java > 8 

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

**Comment by sschuberth** [comment]
@bs-ondem could you please check whether this also works for you locally on Java 8?

**Comment by bs-ondem** [comment]
@sschuberth I tested it with Java 8 and it works 👍

File: reporter/src/main/kotlin/reporters/AntennaAttributionDocumentReporter.kt [hunk 1]
```diff
@@ -84,8 +84,14 @@ class AntennaAttributionDocumentReporter : Reporter {
         var templateId = DEFAULT_TEMPLATE_ID
         options[TEMPLATE_ID]?.let { id ->
             options[TEMPLATE_PATH]?.let { path ->
+                val templatePath = File(path)
```
Comment (mnonnenmacher) [inline reporter/src/main/kotlin/reporters/AntennaAttributionDocumentReporter.kt:91]: Commit message: s/of/if

File: reporter/src/main/kotlin/reporters/AntennaAttributionDocumentReporter.kt [hunk 2]
```diff
@@ -90,19 +94,21 @@ class AntennaAttributionDocumentReporter : Reporter {
...
 
-        val workingDir = createTempDir()
         val documentFile = AttributionDocumentGeneratorImpl(
             defaultFilename,
             workingDir,
             templateId,
             DocumentValues(rootProject.id.name, rootProject.id.version, projectCopyright)
         ).generate(artifacts)
 
+        if (templateId != DEFAULT_TEMPLATE_ID) removeTemplateFromClassPath()
```
Comment (mnonnenmacher) [inline reporter/src/main/kotlin/reporters/AntennaAttributionDocumentReporter.kt:110]: Shouldn't this be in a `finally` block to not risk leaving the Thread with the wrong classloader?

File: reporter/src/main/kotlin/reporters/AntennaAttributionDocumentReporter.kt (hunk 1 above)
Comment (sschuberth) [reply reporter/src/main/kotlin/reporters/AntennaAttributionDocumentReporter.kt:91]: Done.

File: reporter/src/main/kotlin/reporters/AntennaAttributionDocumentReporter.kt (hunk 2 above)
Comment (sschuberth) [reply reporter/src/main/kotlin/reporters/AntennaAttributionDocumentReporter.kt:110]: Yeah, that's a good idea, done. Although keeping the class loader probably wouldn't have done any harm as it's a child class loader of the original class loader.

**Pull request merged**